### ISSUE TITLE
configure: HW_TIMER for aarch64 has wrong explanatory text in config.h

### ISSUE THREAD

**Issue by tonycurtis** [openucx/ucx#5984]
### Describe the bug

Typo in configure re. HW_TIMER setting for aarch64

### Steps to Reproduce

github master @ b9ec32456daff060447921daa896bf6b66b9b276

### Setup and versions
N/A probably

### Additional information (depending on the issue)

In `src/ucs/configure.m4`

```
#
# Architecture specific checks
#
case ${host} in
    aarch64*)
    AC_MSG_CHECKING([support for CNTVCT_EL0 on aarch64])
    AC_RUN_IFELSE([AC_LANG_PROGRAM(
                  [[#include <stdint.h>]],
                  [[uint64_t tmp; asm volatile("mrs %0, cntvct_el0" : "=r" (tmp));]])],
                  [AC_MSG_RESULT([yes])]
                  [AC_DEFINE([HAVE_HW_TIMER], [1], [high-resolution hardware timer enabled])],
                  [AC_MSG_RESULT([no])]
                  [AC_DEFINE([HAVE_HW_TIMER], [0], [high-resolution hardware timer disabled])]
                 );;
    *)
    # HW timer is supported for all other architectures
    AC_DEFINE([HAVE_HW_TIMER], [1], [high-resolution hardware timer **_disabled_**])
esac
```
last AC_DEFINE should have explanatory text "enabled"?

**Comment by shamisp** [comment]
@tonycurtis please open PR.

**Comment by shamisp** [comment]
fixed by #5985
